.travis.yml

```diff
@@ -2,6 +2,9 @@
 language: java
 jdk:
   - oraclejdk8
+env:
+  - SCRIPT=bin/unit-tests.sh
+  - SCRIPT=bin/journey-test.sh
```

Comment:
Cool! This seems to be working great! https://travis-ci.org/exercism/java/builds/251297232